src/scripts/html-transforms.js

```diff
@@ -28,6 +28,15 @@ export function removeStyleAndScript (document) {
   return document;
 }
 
+
+// Add target="_blank" to <a>tags
+export function addTargetBlank (document) {
```

Comment:
Can we add a test in `src/scripts/__tests__/html-transforms.test.js` for this function? Also would make sure that `transforms.reduce` logic doesn't break again in the future 😅

Reply:
Also maybe explain in the function comment why we need to add `target=_blank` to these links in the first place? In case someone new/forgetful comes across this and wonders why we're doing that.

Reply:
👆 Yeah, this function should really have a doc comment like `loadSubresourcesFromWayback` does. It’s a mistake that `removeStyleAndScript` and `compose` are not clearly documented.